src/System.Security.Cryptography.Pkcs/src/Internal/Cryptography/Pal/Unix/DecryptorPalOpenSsl.Decrypt.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security.Cryptography.Pkcs;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Internal.Cryptography.Pal.OpenSsl
+{
+    internal sealed partial class DecryptorPalOpenSsl : DecryptorPal
```

Comment:
While moving this method to a new file might make sense; that seems more related to a future change than this one...